x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/event_correlation_rule.cy.ts

```diff
@@ -174,4 +183,34 @@ describe('EQL rules', { tags: ['@ess', '@serverless'] }, () => {
         });
     });
   });
+
+  describe('with source data requiring EQL overrides', () => {
```

Comment:
This is the one true 🔴 -> 🟢 test in this PR; the rest are just documenting existing behavior.